src/ontology/fbbt-edit.obo

```diff
@@ -125017,6 +125048,17 @@ relationship: receives_synaptic_input_in_region FBbt:00003888 ! lobula plate lay
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-07-15T10:42:07Z
 
+[Term]
+id: FBbt:00058229
+name: high salt-sensing neuron of labellar L-type sensillum
+def: "A gustatory receptor neuron that innervates a L-type sensillum of the labellum and responds to high concentrations of salts (Jaeger et al., 2018)." [FlyBase:FBrf0240353]
+subset: cur
+synonym: "L2 GRN" BROAD [FlyBase:FBrf0208984]
+intersection_of: FBbt:00100044 ! high salt/bitter-sensing neuron
+intersection_of: has_sensory_dendrite_in FBbt:00007411 ! large labellar taste bristle
```

Comment:
add "'capable of' some 'detection of chemical stimulus involved in sensory perception of salty taste'" (subclass of, not intersection)
this will then autoclassify as 'salt-sensing neuron'